presto-main/src/main/java/io/prestosql/security/ViewAccessControl.java

```diff
@@ -74,4 +77,16 @@ public void checkCanGrantExecuteFunctionPrivilege(SecurityContext context, Strin
     {
         return delegate.getColumnMasks(context, tableName, columnName, type);
     }
+
+    private static void wrapAccessDeniedException(Runnable runnable)
+    {
+        try {
+            runnable.run();
+        }
+        catch (AccessDeniedException e) {
+            String prefix = AccessDeniedException.PREFIX;
```

Comment:
nit: I would inline it (up to you, no change request)

Reply:
I've considered this as well, but I see two points against it:
1. it will be used in several places, so no 'link' between these usages.
2. `AccessDeniedException.PREFIX` is too long. Once static imported - it's not explicit what does `PREFIX` mean.
Will left it as is.